frontend/src/component/changeRequest/ChangeRequest/Changes/Change/strategy-change-diff-calculation.ts

```diff
@@ -0,0 +1,82 @@
+import { IChangeRequestUpdateStrategy } from 'component/changeRequest/changeRequest.types';
+import { IFeatureStrategy } from 'interfaces/strategy';
+import isEqual from 'lodash.isequal';
+
+const hasJsonDiff = (object: unknown, objectToCompare: unknown) =>
+    JSON.stringify(object) !== JSON.stringify(objectToCompare);
```

Comment:
This function also exists as `hasDiff` in the StrategyChange.tsx component, but I'm not sure they'll always change at the same rate, so I'm copying it in here slightly renamed.

Reply:
Why is there ported to the BE?